modules/web-console/frontend/app/components/page-queries/components/queries-notebook/controller.ts

```diff
@@ -963,19 +963,24 @@ export class NotebookCtrl {
             const cluster$ = agentMgr.connectionSbj.pipe(
                 pluck('cluster'),
                 distinctUntilChanged(),
-                tap((cluster) => {
-                    this.clusterIsAvailable = (!!cluster && cluster.active === true) || agentMgr.isDemoMode();
+            );
+
+            const checkState$ = combineLatest(
+                cluster$,
+                agentMgr.currentCluster$.pipe(pluck('state'), distinctUntilChanged())
+            ).pipe(
+                tap(([cluster, state]) => {
+                    if (state === 'CONNECTED')
+                        this.clusterIsAvailable = (!!cluster && cluster.active === true) || agentMgr.isDemoMode();
```

Comment:
I can't find any uses of `clusterIsAvailable` 🤷 Maybe remove the `tap` completely?

Reply:
Removed unused variable with tap function.